.node-version

```diff
@@ -1 +1 @@
-14.16.0
+18.0.0
```

Comment:
Updating several major versions seems concerning, what steps were taken to ensure the rest of the codebase was not affected?